src/flows/CostCalculator/hooks.ts

```diff
@@ -99,6 +101,22 @@ const useCostCalculatorEstimation = () => {
   });
 };
 
+export const useCostCalculatorEstimationPdf = () => {
```

Comment:
Can you add a simple JS docs like we have for the other hooks? We'll have to improve this docs some other time 👍